asterisk_mbox_server/__init__.py

```diff
@@ -79,14 +79,16 @@ def _build_cdr(entries, start=0, count=-1, sha=None):
 
     def _send(self, command, msg):
         """Send message prefixed by length."""
+        print('SEND command %s' % (command))
```

Comment:
Please remove all print()  calls.  replace with logging.debug() if needed

Reply:
I will remove the prints with an other pull request.

Reply:
please just update this pull request with the fixes